lib/mobile_workflow/cli/app_builder.rb

```diff
@@ -78,6 +74,15 @@ def generate_dot_env
 ADMIN_PASSWORD=#{admin_password}
 CODE
     end
+
+    def rubocop
+      copy_file '.rubocop.yml'
+      command = 'rubocop --auto-gen-config'
```

Comment:
I think it is OK for now. This is ignoring about 3 type of offenses and I've added them here because I do not want to disable the offense, but it is also not quick to fix now. I need more time to investigate and I do not think it is that much of a priority as other tasks I have. 

One example: I am ignoring the offense FrozenStringLiteral missing in Rails generated files because it is not easy for me to add the string, as files are not created by the generator but by Rails itself. I also don't want to disable the rule because I think it is important, so I prefer to ignore them for now and when creating a project, we fix them manually as it is way faster.